penumbra/src/bin/pd.rs

```diff
@@ -55,7 +55,7 @@ async fn main() {
             let _db_insert_dummy_row = db_insert(
                 PenumbraNoteCommitmentTreeAnchor::from(NoteCommitmentTreeAnchor {
                     id: 0,
-                    height: 1337 as i64,
```

Comment:
Instead of casting, we can write the literal with the appropriate type to begin with